app/controllers/assigned_applications_controller.rb

```diff
@@ -1,14 +1,17 @@
 class AssignedApplicationsController < ApplicationController
-  before_action :set_assigned_application, except: [:index, :create, :next_application]
-
   def index
-    @applications = current_user.assigned_applications
+    current_assignments = CurrentAssignment.where(
+      user_id: current_user_id
+    )
+
+    @applications = current_assignments.pluck(:assignment_id).map { |a_id| CrimeApplication.find(a_id) }
```

Comment:
Tiny thing `a_id` could just be `id` in the block as the fact that it is an assignment id is obvious from the `pluck` :)

Reply:
good call. will fix.